Tests/Storage/FileServiceTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+namespace EMS\CoreBundle\Tests\Service;
+
+use EMS\CommonBundle\Storage\Service\StorageInterface;
+use EMS\CoreBundle\Service\FileService;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use function file_put_contents;
+use function hash_file;
+use function hash_init;
+use function printf;
+use function strlen;
+use function unlink;
+
+
+class FileServiceTest extends WebTestCase
+{
+    public function testStorageServices()
+    {
+        self::bootKernel();
+
+        $this->assertNotNull(self::$container);
+
+        /**@var FileService $fileService */
+        $fileService = self::$container->get('ems.service.file');
```

Comment:
To be verified with David, but I think $container->get is deprecated.

Reply:
Ok don't take into account this remark: it is legit for tests:
https://symfony.com/blog/new-in-symfony-4-1-simpler-service-testing

Reply:
Look at this [post](https://symfony.com/blog/new-in-symfony-4-1-simpler-service-testing). If you have a better solution?